cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -477,15 +481,19 @@ func (s *Server) Index(args *indexArgs) (state indexState, err error) {
 	}
 
 	repositoryName := args.Name
+	repositoryID := args.BuildOptions().RepositoryDescription.ID
 	if _, ok := s.deltaBuildRepositoriesAllowList[repositoryName]; ok {
-		repositoryID := args.BuildOptions().RepositoryDescription.ID
 		debug.Printf("delta build: Server.Index: marking %q (ID %d) for delta build", repositoryName, repositoryID)
-
 		args.UseDelta = true
 	}
 
 	args.DeltaShardNumberFallbackThreshold = s.deltaShardNumberFallbackThreshold
 
+	if _, ok := s.repositoriesSkipSymbolsCalculationAllowList[repositoryName]; ok {
+		debug.Printf("Server.Index: skipping symbols calculation for %q (ID %d)", repositoryName, repositoryID)
```

Comment:
have you been looking at these debug lines? I wonder if they are needed. Personally I'd log these to `tr` if you are not actually checking them out. Otherwise the log lines at the process start reminding you what is set seems the most useful for not forgetting these flags.

Reply:
I switched to logging these in a trace instead. Do we have another way of seeing non-active traces besides Jaeger? 

https://k8s.sgdev.org/-/debug/jaeger - the URL seems to be broken